doc/contributing/releases.md

```diff
@@ -941,6 +941,30 @@ _In whatever form you do this..._
 
 ### Marking a release line as LTS
 
+The process of marking a release line as LTS has been automated using
+[node-core-utils](https://github.com/nodejs/node-core-utils).
+
+Start by checking out the staging branch for the release line that is going to
+be marked as LTS, e.g:
+
+```console
+$ git checkout v1.x-staging
+```
+
+Next, make sure you have **node-core-utils** installed:
+
+```console
+$ npm i -g node-core-utils
+```
+
+Run the prepare LTS release command:
+
+```console
+$ git node release --prepare --startLTS
+```
+
+<details>
+<summary>Manual steps for reference.</summary>
```

Comment:
I think you need to have a blank line here, otherwise the following lines are not treated as markdown, and the ticks are not converted to `<code>` tags (instead of `src/node_version.h`, if displays \`src/node_version.h`).
```suggestion
<summary>Manual steps for reference.</summary>

```